src/api/withdrawFunds.ts

```diff
@@ -1,73 +1,92 @@
-import withdrawHelperInit from "@gnosis.pm/dex-liquidity-provision/scripts/wrapper/withdraw";
+import { amountUSDValue } from "@gnosis.pm/dex-liquidity-provision/scripts/utils/price_utils";
+import { MAXUINT256, ONE } from "@gnosis.pm/dex-liquidity-provision/scripts/utils/constants";
+import { importTradingStrategyHelpers, importWithdrawWrapper } from "api/utils/dexImports";
+import { getWithdrawableAmount } from "@gnosis.pm/dex-contracts";
 import makeFakeArtifacts from "utils/makeFakeArtifacts";
+
+import getLogger from "utils/logger";
+
 import { Web3Context } from 'types';
 import Strategy from 'logic/strategy';
 
-let initializedWithdrawHelpers;
+import BN from "bn.js";
 
-const runInitializerIfNotRan = (context) : any => {
-  if (!initializedWithdrawHelpers) {
-    initializedWithdrawHelpers = withdrawHelperInit(
-      context.instance,
-      makeFakeArtifacts(context)
-    )
-  }
-  return initializedWithdrawHelpers;
-}
+const logger = getLogger("withdraw");
 
-export const withdrawRequest = async (context : Web3Context, strategy : Strategy) : Promise<any> => {
+export const withdrawRequest = async (context : Web3Context, strategy : Strategy) : Promise<void> => {
   const contracts = await Promise.all([
     context.getArtifact("IProxy"),
     context.getArtifact("GnosisSafe"),
     context.getArtifact("MultiSend"),
     context.getArtifact("BatchExchange"),
     context.getArtifact("FleetFactory"),
+    context.getArtifact("GnosisSafeProxyFactory"),
   ]);
-  
-  const withdrawHelpers = runInitializerIfNotRan(context);
-
-  const withdrawTransactions = await withdrawHelpers.prepareWithdrawRequest( // prepareTransferFundsToMaster
-    {
-      masterSafe: context.safeInfo.safeAddress,
-      brackets: strategy.brackets.map(bracket => bracket.address),
-      tokens: [strategy.baseTokenAddress, strategy.quoteTokenAddress],
-    }, true
+
+  const  amountFunction = async function (bracketAddress, tokenData, exchange) : Promise<String> {
+    const amount = (await exchange.getBalance(bracketAddress, tokenData.address)).toString()
+    const usdValue = await amountUSDValue(amount, tokenData, {})
+    logger.log(`(request) ${bracketAddress} holds ${amount} (${usdValue}$) in ${tokenData.symbol}`);
+    if (usdValue.gte(ONE)) {
+      return MAXUINT256.toString()
+    } else {
+      return "0"
```

Comment:
I know this is something from the script and nothing new to the CMM app, but what about tokens that do not have yet a correct price estimation? 
Won't they have the option to bypass this check and force withdrawing anyway?